zircon.jvm.examples/src/main/kotlin/org/hexworks/zircon/examples/fragments/table/TableExample.kt

```diff
@@ -0,0 +1,181 @@
+package org.hexworks.zircon.examples.fragments.table
+
+import org.hexworks.cobalt.databinding.api.binding.bindTransform
+import org.hexworks.cobalt.databinding.api.value.ObservableValue
+import org.hexworks.zircon.api.*
+import org.hexworks.zircon.api.application.AppConfig
+import org.hexworks.zircon.api.builder.data.TileBuilder
+import org.hexworks.zircon.api.color.ANSITileColor
+import org.hexworks.zircon.api.component.HBox
+import org.hexworks.zircon.api.component.Icon
+import org.hexworks.zircon.api.component.Label
+import org.hexworks.zircon.api.component.VBox
+import org.hexworks.zircon.api.data.Position
+import org.hexworks.zircon.api.fragment.Table
+import org.hexworks.zircon.api.graphics.BoxType
+import org.hexworks.zircon.api.graphics.Symbols
+import org.hexworks.zircon.api.screen.Screen
+import org.hexworks.zircon.api.uievent.ComponentEventType
+
+/**
+ * This example shows the usage of the table fragment.
+ */
+object TableExample {
```

Comment:
Can we attach a screenshot of what this looks like to the review somewhere? And/or a gif?

Reply:
Will do.

Reply:
attached it to the issue as I think it is more "historically relevant" as this PR: https://github.com/Hexworks/zircon/issues/185#issuecomment-817187255